examples/ffi/crashinfo.cpp

```diff
@@ -64,13 +65,29 @@ EXTRACT_RESULT(CrashInfo, DDOG_CRASHT_RESULT_HANDLE_CRASH_INFO_OK_HANDLE_CRASH_I
 EXTRACT_RESULT(StackTrace, DDOG_CRASHT_RESULT_HANDLE_STACK_TRACE_OK_HANDLE_STACK_TRACE)
 EXTRACT_RESULT(StackFrame, DDOG_CRASHT_RESULT_HANDLE_STACK_FRAME_OK_HANDLE_STACK_FRAME)
 
-void add_stacktrace(ddog_crasht_Handle_CrashInfoBuilder *builder) {
-  auto stacktrace = extract_result(ddog_crasht_StackTrace_new(), "failed to make new StackTrace");
+std::optional<std::string> demangle(std::string const& name)
+{
+  // We must keep this call to demangle to check that the symbol is exported.
+  auto result = ddog_crasht_demangle(to_slice_string(name), DDOG_CRASHT_DEMANGLE_OPTIONS_COMPLETE);
+  if (result.tag == DDOG_CRASHT_RESULT_STRING_WRAPPER_OK_STRING_WRAPPER)
+  {
+    // TODO: There is currently no safe way to free the StringWrapper
+    auto stringWrapper = result.ok;
+    return std::string((char*)stringWrapper.message.ptr, stringWrapper.message.len);
```

Comment:
This seems like a problem.  Would a different return type be better here?  Or having a `drop` ffi function

Reply:
hum I saw 
`ddog_StringWrapper` and `ddog_crasht_StringWrapper` ....
I will clean it up